File: API_SC_01_Marco2023/G1/prova_rogerio_teresa/consorcio.js
import {question} from 'readline-sync'

function main (){
    const valor_bem= Number(question('Informe o valor do bem em reais:'))
    const prazo= Number(question('Informe o prazo em meses:'))
    const parcel= Number(question('Parcelas em reais:')) 
    const taxa_adm= Number(question('Informe a taxa de administração:'))
    const lance= Number(question('Informe o valor do lance proposto:'))
    const renda= Number(question('Informe sua renda mensal:'))

    const parcela= (valor_bem+ taxa_adm/100 * valor_bem) / prazo
    const total_p= valor_bem+ taxa_adm/100 * valor_bem 
    const total_adm= valor_bem * taxa_adm/100 

    if (lance> 30/100 * renda){
        console.log('Resgate Reprovado')
    } else if (lance<=30/100 * renda){
        console.log('Resgate Aprovado')
        console.log('Nova parcela:', parcela)
        console.log('Novo prazo:', prazo)  
    }
    

 

}
main()

File: API_SC_01_Marco2023/G1/germano junior/consorcio.py

# Germano Júnior 2023111TADS0015 2023-março-27


def main():
    print("TODO: terminar :(")
    # ENTRADA
    valor = float(input("Valor do bem: "))
    prazo = int(input("Prazo (meses): "))
    taxa_adm = float(input("Taxa de administração (em porcentagem): "))
    lance = float(input("Valor lance proposto: "))
    manter = input("manter (prazo ou parcela): ")
    if manter != "prazo" and manter != "parcela":
        print("informe ou prazo ou parcela no campo manter")
        return
    renda = float(input("Renda mensal: "))
    # PROCESSAMENTO
    total_adm = valor * taxa_adm
    total_valor = total_adm + valor
    valor_parcela = 2
    if manter == "prazo":
        valor_parcela = (total_valor - lance) / prazo
    elif manter == "parcela":
        valor_parcela = total_valor / prazo
        prazo = prazo - lance / valor_parcela
    valor_resgate = valor_parcela * 0.30
    # SAÍDA
    print("---------------------------------------------------------")
    print(f"Valor das parcelas: {valor_parcela * prazo}")
    print(f"Total a pagar: {total_valor}")
    print(f"Total taxa administração: {total_adm}")
    if renda < valor_resgate:
        print("É possível resgatar o bem")
    else:
        print("Não é possível resgatar o bem")
        print(f"Renda mínima para o resgate: {valor_resgate}")


if __name__ == "__main__":
    main()
